### DISCUSSION TITLE
makepkg error

### DISCUSSION THREAD

**Discussion by daviardev** [prasanthrangan/hyprdots#365] [General]
When I run the ./install.sh, it loads everything, but then I get an error in the makepkg, it says you can not run as superuser mode is not allowed as it may cause permanent t catastrophic damage to your system.

I enter as super user as it does not allow me to access in normal mode.

**Comment by prasanthrangan** [comment]
Hi, please do NOT execute the install script as sudo or root user... it will ask you for sudo password only when its absolutely required.
